### DISCUSSION TITLE
v.1.0.0: now no need to add div  container manually. Leave every things for th…

### DISCUSSION THREAD

**Discussion by saarock** [saarock/notty#23] [Announcements]
First version release for production.

<hr /><em>This discussion was created from the release <a href='https://github.com/saarock/notty.js/releases/tag/v.1.0.0'>v.1.0.0: now no need to add div  container manually. Leave every things for th…</a>.</em>

**Comment by angeldevildev** [comment]
> First version release for production.
> 
> _This discussion was created from the release [v.1.0.0: now no need to add div container manually. Leave every things for th…](https://github.com/saarock/notty.js/releases/tag/v.1.0.0)._

Ok, got it 👍
